src/models/PendingTalkClaimMapper.php

```diff
@@ -167,4 +167,40 @@ public function approveClaimAsHost($talk_id, $speaker_id, $claim_id)
         );
         return $response;
     }
+
+    public function getPendingClaimsByEventId($event_id)
+    {
+        $base       = $this->_request->base;
+        $version    = $this->_request->version;
+
+        $sql = 'select c.*, s.speaker_name from pending_talk_claims c
+                    inner join talks t on t.ID = c.talk_id
+                    inner join talk_speaker s on s.ID = c.claim_id
+                    where t.event_id = :event_id and
+                    (host_approved_at IS NULL or user_approved_at IS NULL)';
+        $stmt = $this->_db->prepare($sql);
+        $response = $stmt->execute(['event_id' => $event_id]);
+
+        if ($response && $stmt->rowCount() > 0) {
```

Comment:
I'd alter this to
 ```php
if (! $response || $stmt->rowCount() < 1) {
    return false;
}
```
and then continue with the stuff you currently have in the else.

That way the "default flow" is going through linearily and exceptions fail fast